crates/revm/src/db/states/bundle_state.rs

```diff
@@ -296,6 +306,7 @@ impl BundleState {
 
     /// Extend the state with state that is build on top of it.
     ///
+    // mattsse: this paragraph is very hard to parse:
```

Comment:
reworded it here: https://github.com/bluealloy/revm/commit/28976553c1110e943c6adfb65a22fd878c957ef4